### PR TITLE
Add serviceable attribute to projects.

### PR DESCRIPTION
**Pull request by NTaylorMullen**

### CODE REVIEW

File: src/Microsoft.AspNet.JsonPatch/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (pranavkm) [inline src/Microsoft.AspNet.JsonPatch/Properties/AssemblyInfo.cs]: Out of curiosity, is the casing for `True` supposed to look like this?

**Comment by pranavkm** [comment]
:shipit:

File: src/Microsoft.AspNet.Mvc.Razor.Host/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -1,6 +1,8 @@
-﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```
Comment (dougbu) [inline src/Microsoft.AspNet.Mvc.Razor.Host/Properties/AssemblyInfo.cs]: is this another BOM change?

File: src/Microsoft.AspNet.JsonPatch/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (Eilon) [inline src/Microsoft.AspNet.JsonPatch/Properties/AssemblyInfo.cs]: Yes

**Comment by Eilon** [comment]
:shipit:

File: src/Microsoft.AspNet.Mvc.Razor.Host/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (NTaylorMullen) [inline src/Microsoft.AspNet.Mvc.Razor.Host/Properties/AssemblyInfo.cs]: Actually we're removing the BOM. Found that our assemblyinfo.cs files are extreeemly inconsistent (almost seems like a 50/50 split) with their bom's, decided to remove all of them.

**Pull request merged**